src/index.js

```diff
@@ -4,8 +4,17 @@ import { Provider } from "react-redux";
 import store from "../src/js/store/index";
 import App from "./containers/index";
 
+// Firebase App (the core Firebase SDK) is always required and
+// must be listed before other Firebase SDKs
+var firebase = require("firebase/app");
```

Comment:
@magicmarie These firebase lines should be in another firebase js file with the Firebase configs